activity.py

```diff
@@ -69,28 +85,70 @@ def on_button_clicked(self, button, button_number):
         self.start_game(button_number)
 
     def create_game_screen(self, level):
-        # Loading image and creating pixbuf
+        overlay = self.create_overlay_from_image(
+            GAME_BACKGROUND_PATH.format(level))
+
+        self.create_text_view()
+        self.create_entry_field()
```

Comment:
It'll be better to move the definitions of `create_text_view` and `create_entry_field` to right after this function as it's better code organization.

Do same with `create_toolbar` and `create_menu` above, as they're called at the end of the init function.

Reply:
Yep, makes sense